lessons/lesson1.md

```diff
@@ -1,5 +1,16 @@
-## Lesson 1: Bootstrap
+## Lesson 1: Setup and Overview
```

Comment:
I feel that we should jump directly to the "step by step" guide instead of repeating setup and overview which was already covered in intro and requirements.

Reply:
good point, reduce from 3 lessons to 2 lessons